Modules/_testinternalcapi.c

```diff
@@ -35,6 +36,38 @@ get_recursion_depth(PyObject *self, PyObject *Py_UNUSED(args))
 }
 
 
+static PyObject*
+codecs_unregister(PyObject *self, PyObject *args)
+{
+    PyObject *search_function;
+
+    if (!PyArg_ParseTuple(args, "O:codecs_unregister",
+                          &search_function)) {
+        return NULL;
+    }
+    if (search_function == NULL) {
+        return NULL;
```

Comment:
It cannot be NULL.

Reply:
Thanks, You are right, I will update it.